apps/src/templates/amazonFutureEngineerEligibility/amazonFutureEngineerEligibilityForm.jsx

```diff
@@ -273,6 +316,43 @@ export default class AmazonFutureEngineerEligibilityForm extends React.Component
                     : null
                 }
               />
+              <p>Tell CSTA a little bit about yourself:</p>
+              <label
```

Comment:
Can you post an image of the form in the PR description? I think it would make it easier for me to have an overall handle on the labeling!

Reply:
For some reason, GitHub was blocking my uploads, but I added one to the Jira board. Let me see if I can find another work around . . .

Reply:
Still no luck. Does the Jira image work?

Reply:
Oh yeah for sure! I'll look there!

Reply:
I pulled the branch locally and made it to http://localhost.code.org:3000/afe. Just looking to fill out a form with a screen reader as a proxy for reviewing all these nice label tags! Do you know how I can actually get to this form?

Reply:
I'm having trouble seeding schools locally so that this school shows up in the dropdown. To not hold you back any longer from merging, I'm happy to take another look when it's up on staging and then I can always open a tiny followup pr if I happen to find any accessibility notes!